src/app/record/components/affiliation/affiliation.component.html

```diff
@@ -9,24 +9,22 @@
     <div class="general-data">
       <ng-container *ngIf="affiliation?.startDate?.year"
         >{{ affiliation.startDate | monthDayYearDateToString }}
-        <ng-container> to </ng-container>
-        <ng-container *ngIf="affiliation.endDate">{{
-          affiliation.endDate | monthDayYearDateToString
-        }}</ng-container>
-        <ng-container *ngIf="!affiliation.endDate.year">present</ng-container>
-      </ng-container>
-      <ng-container
-        *ngIf="affiliation?.startDate.year && affiliation?.roleTitle.value"
-      >
-        |
+        <ng-container *ngIf="showEndDate">
+          <ng-container> to </ng-container>
+          <ng-container *ngIf="affiliation.endDate">{{
+            affiliation.endDate | monthDayYearDateToString
+          }}</ng-container>
+          <ng-container *ngIf="!affiliation.endDate.year">present</ng-container>
```

Comment:
@leomendoza123 I hadn't realized that we don't have i18n attributes

Reply:
got catch fixing